lib/decidim.rb

```diff
@@ -23,3 +23,4 @@
 require "decidim/debates"
 require "decidim/sortitions"
 require "decidim/blogs"
+require "decidim/design"
```

Comment:
Same here, as this is not a default module (i.e. it's an opt-in commented out by default in the `Gemfile`) then this should not be here. You need to check out where the modules that have this same behavior are (i.e. decidim-conferences, -elections, -initiatives)

```suggestion
```